### DISCUSSION TITLE
Chat widget modifications?

### DISCUSSION THREAD

**Discussion by Jarrodsz** [chatwoot/chatwoot#5060] [Q&A]
**Is your enhancement request related to a problem? Please describe.**

Need some modifications to the chat widget. Since its build in next would it be possible to custom build the widget or can the widget be placed in a seperate coddebase? I need it to look more like intercom no full screen widget expansion etc.

**Describe the solution you'd like**

**Comment by pranavrajs** [comment]
@Jarrodsz This feels like a discussion topic rather than an issue.  Can you please share the changes you are trying to do and what is blocking at the moment?
